data/intro missions.txt

```diff
@@ -382,7 +382,7 @@ mission "Intro [2 Interceptor]"
 				`	"Got it."`
 					accept
 				`	"I really shouldn't try to fight?"`
-			`	"It just isn't feasible," he says. "If a ship has turrets, your Sparrow can't get close enough to hit it without being damaged, and then it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it."`
+			`	"It just isn't feasible," he says. "If a ship has gun turrets, your Sparrow can't get close enough to hit it with your own fixed guns without being damaged. If it has anti-missile turrets, you can't count on hanging back and bombarding it with missiles from long distance - at least not with weapons can fit in a Sparrow. In this case it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it."`
```

Comment:
`at least not with weapons can fit in a Sparrow.` -> ` at least not with weapons that can fit in a Sparrow.` ?